lib/vanagon/platform/deb.rb

```diff
@@ -21,9 +21,9 @@ def generate_package(project)
       # Method to generate the files required to build a debian package for the project
       #
       # @param workdir [String] working directory to stage the evaluated templates in
-      # @param name [String] name of the project
+      # @param project [Vanagon::Project] project to build a debian package of
       # @param binding [Binding] binding to use in evaluating the packaging templates
-      def generate_packaging_artifacts(workdir, name, binding)
```

Comment:
@melissa Debated about that one -
Originally changed it to ```def generate_packaging_artifacts(project, workdir, name, binding)```
But then thought - its now 4 arguments, and ```project``` already includes ```"name"```
If you look at the ```generate_package_method``` you will see that ```project``` is passed and ```name``` is referenced, so I thought it was a more consistent way to do it.